2021-10-12 19:21:06,708 INFO    MainThread:3036 [wandb_setup.py:_flush():69] setting env: {}
2021-10-12 19:21:06,708 INFO    MainThread:3036 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-12 19:21:06,709 INFO    MainThread:3036 [wandb_init.py:_log_setup():342] Logging user logs to c:\Users\reese\OneDrive\Documents\University\EEE4022S\Workspace\Final-Year-Project\wandb\run-20211012_192106-18g6k1ds\logs\debug.log
2021-10-12 19:21:06,709 INFO    MainThread:3036 [wandb_init.py:_log_setup():343] Logging internal logs to c:\Users\reese\OneDrive\Documents\University\EEE4022S\Workspace\Final-Year-Project\wandb\run-20211012_192106-18g6k1ds\logs\debug-internal.log
2021-10-12 19:21:06,709 INFO    MainThread:3036 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002BA199C01F0>
2021-10-12 19:21:06,711 INFO    MainThread:3036 [wandb_init.py:init():375] calling init triggers
2021-10-12 19:21:06,711 INFO    MainThread:3036 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-12 19:21:06,711 INFO    MainThread:3036 [wandb_init.py:init():424] starting backend
2021-10-12 19:21:06,711 INFO    MainThread:3036 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-10-12 19:21:06,717 INFO    MainThread:3036 [backend.py:ensure_launched():135] starting backend process...
2021-10-12 19:21:06,810 INFO    MainThread:3036 [backend.py:ensure_launched():139] started backend process with pid: 3792
2021-10-12 19:21:06,811 INFO    MainThread:3036 [wandb_init.py:init():429] backend started and connected
2021-10-12 19:21:06,875 INFO    MainThread:3036 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-10-12 19:21:06,877 INFO    MainThread:3036 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-12 19:21:06,877 INFO    MainThread:3036 [wandb_init.py:init():477] updated telemetry
2021-10-12 19:21:06,878 INFO    MainThread:3036 [wandb_init.py:init():500] communicating current version
2021-10-12 19:21:08,155 INFO    MainThread:3036 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-12 19:21:08,156 INFO    MainThread:3036 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-12 19:21:08,524 INFO    MainThread:3036 [wandb_init.py:init():540] starting run threads in backend
2021-10-12 19:21:08,611 INFO    MainThread:3036 [wandb_run.py:_console_start():1601] atexit reg
2021-10-12 19:21:08,612 INFO    MainThread:3036 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-10-12 19:21:08,612 INFO    MainThread:3036 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-10-12 19:21:08,615 INFO    MainThread:3036 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-12 19:21:08,615 INFO    MainThread:3036 [wandb_init.py:init():565] run started, returning control to user process
2021-10-12 19:21:08,619 INFO    MainThread:3036 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-12 19:21:08,619 INFO    MainThread:3036 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-12 19:21:08,730 INFO    MainThread:3036 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-12 19:21:08,732 INFO    MainThread:3036 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-12 19:21:08,732 INFO    MainThread:3036 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-12 19:21:08,788 INFO    MainThread:3036 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-12 19:21:08,789 INFO    MainThread:3036 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-12 19:21:08,789 INFO    MainThread:3036 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-12 19:21:08,826 INFO    MainThread:3036 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-12 19:25:15,665 INFO    MainThread:3036 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-12 19:25:15,665 INFO    MainThread:3036 [jupyter.py:save_ipynb():298] not saving jupyter notebook
